### PR TITLE
Adding BVG, VBB and S-Bahn Berlin. Updated DB

### PR DESCRIPTION
**Pull request by dschemp**
BVG is the main public transport provider in Berlin complemented by the S-Bahn Berlin GmbH. The VBB is the transport association for public transport in Berlin and Brandenburg.

### CODE REVIEW

File: companies/bvg.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    "name": "Berliner Verkehrsbetriebe (BVG)",
+    "runs": [
+        "URBANIS GmbH",
+        "BT Berlin Transport GmbH (BT)",
+        "IOB Internationale Omnibusbahnhof-Betreibergesellschaft mbH",
+        "BVG Projekt GmbH",
+        "BVG Beteiligungsholding Verwaltungsgesellschaft mbH",
+        "BVG Beteiligungsholding GmbH & Co. KG"
+    ],
+    "address": "Berliner Verkehrsbetriebe (BVG) - AöR -\nDatenschutzbeauftragter\nHolzmarktstraße 15-17\n10179 Berlin\nDeutschland",
```
Comment (mal-tee) [inline companies/bvg.json:19]:
```suggestion
    "address": "Holzmarktstraße 15-17\n10179 Berlin\nDeutschland",
```
Our generator adds the company line and the DSB, so we do not need it in the address.

File: companies/sbahn-berlin.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{
+    "slug": "sbahn-berlin",
+    "relevant-countries": [
+        "de"
+    ],
+    "categories": [
+        "travel"
+    ],
+    "name": "S-Bahn Berlin GmbH",
+    "address": "S-Bahn Berlin GmbH\nDatenschutz\nElisabeth-Schwarzhaupt-Platz 1\n10115 Berlin\nDeutschland",
```
Comment (mal-tee) [inline companies/sbahn-berlin.json:10]:
```suggestion
    "address": "Elisabeth-Schwarzhaupt-Platz 1\n10115 Berlin\nDeutschland",
```

File: companies/sbahn-berlin.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    "phone": "+49 30 29743333",
+    "email": "Datenschutz.Regio@deutschebahn.com",
+    "sources": [
+        "https://sbahn.berlin/datenschutz/",
+        "https://sbahn.berlin/impressum/"
+    ],
+    "comments": [
+        "Die bestellte Datenschutzbeauftragte der S-Bahn Berlin GmbH ist Frau Chris Newiger.",
```
Comment (mal-tee) [inline companies/sbahn-berlin.json:18]:
```suggestion       
```
We don't want to have any personal data, including names, in our database.

File: companies/vbb.json [hunk 4]
```diff
@@ -0,0 +1,22 @@
+{
+    "slug": "vbb",
+    "relevant-countries": [
+        "de"
+    ],
+    "categories": [
+        "travel"
+    ],
+    "name": "VBB Verkehrsverbund Berlin-Brandenburg GmbH",
+    "address": "VBB Verkehrsverbund Berlin-Brandenburg GmbH\nDatenschutz\nHardenbergplatz 2\n10623 Berlin\nDeutschland",
```
Comment (mal-tee) [inline companies/vbb.json:10]:
```suggestion
    "address": "Hardenbergplatz 2\n10623 Berlin\nDeutschland",
```

File: companies/vbb.json [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+    "email": "datenschutz@vbb.de",
+    "web": "https://www.vbb.de/",
+    "sources": [
+        "https://www.vbb.de/datenschutz"
+    ],
+    "comments": [
+        "Fragen oder Anregungen zur Datenverarbeitung beim VBB können auch per E-Mail an info@vbb.de oder telefonisch an +49 30 254140 gestellt werden."
```
Comment (mal-tee) [inline companies/vbb.json:17]: You can move the telephone number into the `"phone"` field

**Review by mal-tee — CHANGES_REQUESTED** [review]
Thank you very much! I've left some minor requests for changes.

Please note that this is my first review in `data`, @baltpeter might have some more comments.

File: companies/sbahn-berlin.json [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+        "Fragen oder Anregungen zur Datenverarbeitung bei der S-Bahn Berlin GmbH können auch an kundenbetreuung@s-bahn-berlin.de gestellt werden.",
+        "Die S-Bahn Berlin GmbH ist eine Tochtergesellschaft der DB Regio AG."
+    ],
+    "request-language": "de",
+    "quality": "verified"
+}
```
Comment (mal-tee) [inline companies/sbahn-berlin.json:24]:
```suggestion
}

```
The files should end with a newline.

File: companies/vbb.json (hunk 5 above)
Comment (dschemp) [reply companies/vbb.json:17]: The problem in this (as I understood it) is that the phone number is not directly for personal data protection / GDPR request matters.

Comment (baltpeter) [reply companies/vbb.json:17]: That's fine, if there is no phone number specific to data protection matters, we just fallback to a generic one.

File: companies/sbahn-berlin.json [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+    ],
+    "name": "S-Bahn Berlin GmbH",
+    "address": "Elisabeth-Schwarzhaupt-Platz 1\n10115 Berlin\nDeutschland",
+    "phone": "+49 30 29743333",
+    "email": "Datenschutz.Regio@deutschebahn.com",
+    "sources": [
+        "https://sbahn.berlin/datenschutz/",
+        "https://sbahn.berlin/impressum/"
+    ],
+    "comments": [
```
Comment (baltpeter) [inline companies/sbahn-berlin.json:17]: The comments are displayed fairly prominently in the generator, so we try to only include important ones. I think we can remove both of them in this case.

File: companies/sbahn-berlin.json [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+    "relevant-countries": [
+        "de"
+    ],
+    "categories": [
+        "travel"
+    ],
+    "name": "S-Bahn Berlin GmbH",
+    "address": "Elisabeth-Schwarzhaupt-Platz 1\n10115 Berlin\nDeutschland",
+    "phone": "+49 30 29743333",
+    "email": "Datenschutz.Regio@deutschebahn.com",
```
Comment (baltpeter) [inline companies/sbahn-berlin.json:12]: Since this is a data protection-specific email, please set `suggested-transport-medium` to `email`.

**Review by baltpeter — CHANGES_REQUESTED** [review]
Thanks from me also. :)

**Review by baltpeter — APPROVED** [review]
Thanks again.

**Pull request merged**